website/src/controllers/api-v3/groups.js

```diff
@@ -123,6 +123,10 @@ api.getGroups = {
       return previousValue.concat(Array.isArray(currentValue) ? currentValue : [currentValue]); // otherwise concat the new results to the previousValue
     }, []);
 
+    results = _.uniq(results, (group) => {
```

Comment:
@TheHollidayInn why uniq? When it can result in the same challenge returned twice?